bin/ci

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash --login
```

Comment:
`rvm use` didn't work without it in an earlier permutation. Just double checked: https://circleci.com/gh/codeclimate/ruby-test-reporter/135 and still think it's needed for the ruby switching to work